tests/test_tasking.py

```diff
@@ -108,3 +116,94 @@ def test_decide_quotations_live(tasking_live):
         tasking_live.decide_quotation(accepted_quotation_id, "ACCEPTED")
     assert isinstance(e.value, requests.exceptions.RequestException)
     assert "405" in str(e.value)
+
+
+def test_get_feasibility(tasking_get_feasibility_mock):
```

Comment:
@janchrizz will request for more tests, I am sure. Think about the parameters that accept arrays (if there is any).